styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
+        border-radius: 16px;
+        padding: 3rem 4rem;
```

Comment:
https://stackoverflow.com/questions/45822322/when-to-choose-rem-over-px

Reply:
Just trying to make sure what you meant:
"Now, You can use rem for padding and margin etc but it is not recommended because e.g. you have set the rem values of padding and margins and widths etc for your site where your browser default font-size is set to 16 px....So that is why for responsive sites, the units %, vw/vh are mostly used. As for the px unit. They are their to help set static values so that you fix the size rather than have it readjust."
Sums up to: 
**margins, paddings - preferably %, vw, vh for sake of responsive, fixing if needed px
fonts - rem, em**
?

Reply:
But for sake of pixel perfect, or near pp, in most cases px unit is the most common, or?